src/style.css

```diff
@@ -1,3 +1,90 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: gray;
+  text-transform: uppercase;
+
+  --blue: rgba(0, 172, 220, 1);
+  --white: rgba(255, 255, 255, 1);
 }
+
+.header {
+  margin: 0 auto;
+  background-color: var(--white);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__image {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto 10px;
+
+}
+
+.nav__item-first {
+  margin-left: 0;
+}
+
+.nav__item-last {
+  margin-right: 0;
+}
```

Comment:
1. modificators should have 2 `-`, so class shopuld be `nav__item--first`
2. no need to use those modificators, because you can use selector which defines if element is not last, please use `:not(:first-child)` or `:not(:last-child)` depending on which side you will add margin